Gutenberg.podspec

```diff
@@ -0,0 +1,26 @@
+package = JSON.parse(File.read(File.join(__dir__, 'packages', 'react-native-editor', 'package.json')))
```

Comment:
Should we move this file to `@wordpress/react-native-bridge`, or could we even?

Reply:
I tried to move it to `packages/react-native-bridge` but couldn't make it works with `Wordpress-iOS`.  I don't know why but cocoapods can not read `package.json` with that configuration.